gradle.properties

```diff
@@ -2,13 +2,13 @@ org.gradle.caching=true
 group=org.ballerinax.azure.servicebus
 version=3.7.0
 
-puppycrawlCheckstyleVersion=10.12.1
-checkstyleToolVersion=10.12.1
-githubSpotbugsVersion=5.0.14
-githubJohnrengelmanShadowVersion=8.1.1
-underCouchDownloadVersion=5.4.0
-researchgateReleaseVersion=2.8.0
-testngVersion=7.6.1
+checkstylePluginVersion=10.12.0
+spotbugsPluginVersion=5.0.14
+shadowJarPluginVersion=8.1.1
+downloadPluginVersion=5.4.0
+releasePluginVersion=2.8.0
+ballerinaGradlePluginVersion=2.1.5
+jacocoVersion=0.8.10
```

Comment:
Are we using jacoco here?

Reply:
No it was a mistake. Addressed in [d83f05f](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/205/commits/d83f05f2a180df4e2fa1dd36ee1b277ff7cda061)